doc/manual/50-feature-details.en.md

```diff
@@ -237,33 +224,32 @@ rules:
 
 #### Current limitations on backup from standby
 
-Barman currently requires that backup data (base backups and WAL files)
-come from one server only. Therefore, in case of backup from a
-standby, you should point to the standby server:
+Barman currently requires that WAL files and backup data come from the
+same PostgreSQL server. When taking backups from a standby it is therefore
+important to point the following Barman [configuration options][config-options]
+to the standby server:
 
 - `conninfo`
-- `streaming_conninfo`, if you use `postgres` as `backup_method` and/or rely on WAL streaming
-- `ssh_command`, if you use `rsync` as `backup_method`
-
-> **IMPORTANT:** From Barman 2.8, backup from a standby is supported
-> only for PostgreSQL 9.4 or higher (versions 9.4 and 9.5 require
-> `pgespresso`). Support for 9.2 and 9.3 is deprecated.
-
-The recommended and simplest way is to setup WAL streaming
-with replication slots directly from the standby, which requires
-PostgreSQL 9.4. This means:
-
-* configure `streaming_archiver = on`, as described in the "WAL streaming"
-  section, including "Replication slots"
-* disable `archiver = on`
-
-Alternatively, from PostgreSQL 9.5 you can decide to archive from the
-standby only using `archive_command` with `archive_mode = always` and
-by disabling WAL streaming.
-
-> **NOTE:** Unfortunately, it is not currently possible to enable both WAL archiving
-> and streaming from the standby due to the way Barman performs WAL duplication
-> checks and [an undocumented behaviours in all versions of PostgreSQL](https://www.postgresql.org/message-id/20170316170513.1429.77904@wrigleys.postgresql.org).
+- `streaming_conninfo` (when using `backup_method = postgres` or `streaming_archiver = on`)
+- `ssh_command` (when using `backup_method = rsync`)
+
+In the case that the standby is promoted to primary the backups and WALs will
+continue to be valid however you may wish to update the Barman configuration
+so that it uses the new standby for taking backups and receiving WALs.
+
+WALs can be obtained from the standby using either WAL streaming or WAL
+archiving. To use WAL streaming follow the instructions in the
+[WAL streaming](#wal-streaming) section.
+
+To use WAL archiving from the standby follow the instructions in the [WAL archiving via archive_command](#wal-archiving-via-archive_command) section
+*and additionally* set `archive_mode = always` in the PostgreSQL config on
+the standby server.
+
+> **NOTE:** With PostgreSQL 10 and earlier Barman cannot handle WAL streaming
```

Comment:
It's worth adding the full context to the commit message in the unlikely event that someone wants more information: Basically the issue was fixed as a side-effect of https://github.com/postgres/postgres/commit/4a33bb59dfc33566f04e18ab5e1f90b8e7461052 which landed in PG11 - more details can be found at https://gist.github.com/mikewallace1979/3b52cd87faf9a302fefc72377e35fafb.